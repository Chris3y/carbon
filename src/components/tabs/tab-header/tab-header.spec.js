import React from 'react';
import TestUtils from 'react-dom/test-utils';
import TestRenderer from 'react-test-renderer';
import 'jest-styled-components';
import { shallow, mount } from 'enzyme';
import TabHeader from './tab-header.component';
import { elementsTagTest } from '../../../utils/helpers/tags/tags-specs/tags-specs';

function render(props) {
  return TestRenderer.create(
    <TabHeader title='Tab Title 1' id='uniqueid1' {...props}/>
  );
}

describe('TabHeader', () => {
  let wrapper;
  it('renders as expected', () => {
    expect(render()).toMatchSnapshot();
  });
});
