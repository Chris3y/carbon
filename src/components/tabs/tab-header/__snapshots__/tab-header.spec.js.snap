// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabHeader renders as expected 1`] = `
.c0 {
  background-color: transparent;
  border-bottom: 2px solid #E5EAEC;
  color: rgba(0,0,0,0.55);
  display: inline-block;
  font-weight: bold;
  height: 100%;
  margin-left: 2px;
  padding: 11px 15px 10px;
}

.c0:first-child {
  margin-left: 0;
}

.c0:focus,
.c0:hover {
  background: transparent;
  border-bottom-color: #006300;
  color: rgba(0,0,0,0.9);
  outline: none;
}

.c0:focus {
  outline: none;
}

<li
  className="c0"
  data-component="tab-header"
  data-element="select-tab"
  id="uniqueid1"
  role="tab"
>
  Tab Title 1
</li>
`;
